server/routes/probationPractitionerReferrals/probationPractitionerReferralsController.ts

```diff
@@ -59,7 +60,11 @@ export default class ProbationPractitionerReferralsController {
     const { accessToken } = res.locals.user.token
 
     const existingDraftReferrals = await this.interventionsService.getDraftReferralsForUserToken(accessToken)
-    const presenter = new FindStartPresenter(existingDraftReferrals)
+
+    const downloadFileInfo = await FileUtils.fileInformation(
```

Comment:
Can we find a way to avoid the duplication of this filename between here and `FindStartPresenter`?